doc/source/serve/tutorials/batch.md

```diff
@@ -33,9 +33,9 @@ async def my_batch_handler(self, requests: List):
     pass
 ```
 
-This batch handler can then be called from another `async def` method in your deployment.
-These calls will be batched and executed together, but return an individual result as if
-they were a normal function call:
+Ray Serve can then call the batch handler from another `async def` method in your deployment.
+Ray Serve batches and executes these calls together, but returns an individual result as if
+they were a single function call:
```

Comment:
@shrekris-anyscale - is this rewrite technically accurate?

Reply:
Not quite– Ray Serve doesn't call the batch handler from another `async def` method. The user's code calls the batch handler.

Reply:
reverted